api/main.tscn

```diff
@@ -0,0 +1,45 @@
+[gd_scene load_steps=2 format=2]
```

Comment:
This main scene is needed since we are using this scene as the GUI for Gut testing locally.